app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,18 @@
+module SessionsHelper
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id]) if session[:user_id]
+  end
+
+  def logged_in?
+    !current_user.nil?
```

Comment:
Tìm hiểu và cho chị bjk sự khác nhau của nil? và present?

Reply:
Dạ. Cả hai đều dùng cho object, nhưng nil? là check nil của object,trả về true nếu object nil. Còn present? là check Not NIl, Not false, Not empty, trả về true nếu object đó Not Nil, Not false, Not empty.